.gitignore

```diff
@@ -9,3 +9,5 @@ bin/
 
 # Generated files for local development
 local-development/
+
+.idea/
```

Comment:
In general, I like to leave IDE-specific ignore statements out of my project and have devs add them to their global ignore file (https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) but whatevs.

Reply:
I can do that no worries, I don't have a problem with that